Models/NHES/Systems/PrimaryHeatSystem/SMR_Generic/Control_Rod_Model_Simple.bak-mo

```diff
@@ -0,0 +1,53 @@
+within NHES.Systems.PrimaryHeatSystem.SMR_Generic;
```

Comment:
.bak-mo file. Shouldn't be here. Except, based on my brief look, there should be a Control_Rod_Model_Simple file. Is there a .mo file?

Reply:
Taken out.